### ISSUE TITLE
dns_name missing from postgresql module outputs

### ISSUE THREAD

**Issue by steven-edgar** [terraform-google-modules/terraform-google-sql-db#524]
### TL;DR

Add dns_name to outputs of postgresql module

### Expected behavior

dns_name value outputted from postgresql module.
Note: dns_name is not in the documentation for the google_sql_database_instance terraform resource, this has been reported separately via issue (https://github.com/hashicorp/terraform-provider-google/issues/16176)

### Observed behavior

No dns_name output

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
n/a
```

### Additional information

Append the following to modules/postgresql/outputs.tf :

`
output "dns_name" {
  value = module.postgresql-db.default.dns_name
  description = "DNS name of the instance endpoint"
}
`
